Topics/Software_Tools/VSCode.md

```diff
@@ -167,6 +174,88 @@ Here are some quick hand-picked tips and tricks:
+
+<img width="800" alt="Clone Git repo new window" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/416d94f7-23f8-467f-af80-d6a622985a78">
+
+This will make a text box appear. 
+
+<img width="800" alt="Clone textbox" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/b24e534f-8ff9-4b34-960f-458ce2326869">
+
+You can search for the relevant repository and select it. After selecting it, you need to specify a location for the repo. You can either create a new folder or clone it in an existing folder. After selecting the location, you can open the folder and work on the repo!
+
+<img width="800" alt="Git clone select repo location" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/6e6b16e1-05ec-4cf4-987e-3c0e0645bd0b">
+
+
+### Tracking changes
+
+VS Code provides an easy way to track your changes. In the image below I have added changes to the current file. VS Code indicates this with a green bar on all the lines that have been added. If existing lines have been modified, this is indicated with a striped blue bar.
+
+<img width="800" alt="Track changes blue striped bar" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/421a7b5b-bf66-452f-8cac-04f22664aac0">
+
+Click on the blue bar beside the code to look at the modifications that have been made. If you want to revert back to the changes, press the revert button.
+
+<img width="800" alt="Track changes by section" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/711d0053-a7d3-4862-8418-d650dc7b59e9">
+
+You can also look at the changes you've made to the entire file by navigating to the "Source Control" tab on the left pane and clicking on the file.
+
+<img width="800" alt="Track changes whole file" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/0fd5d3f9-f99e-41cf-b0c1-fed50ef362d1">
+
+To look at the previous versions of your file, navigate to the "Explorer" tab on the left pane (if that's not the current tab you are on) and then look at "Timeline" at the bottom of the tab.
+
+<img width="369" alt="Timeline" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/cf2154ac-05fe-4a9c-8c4c-ee527b250d04">
+
+### Adding files to be committed
+
+When you're working on a file, VS Code shows you the status of your files. For example, if you create a new file in your repo, the character 'U' appears beside your file as shown indicating that the file is untracked. 
+
+<img width="363" alt="Before staging untracked" src="https://github.com/vazevaru/learning-software-engineering.github.io/assets/90367491/c8483106-9e08-4c2b-a9c5-a16b276d508e">
+
+If you modify a file existing in the git repo, the character 'M' appears beside your file indicating that it has been modified.
```

Comment:
And their color changes! Mention that!

Reply:
For some reason on my end, in all 3 cases, the color is gray